front/app/containers/Admin/projects/project/analysis/index.tsx

```diff
@@ -0,0 +1,30 @@
+import { Box, colors } from '@citizenlab/cl2-component-library';
+import React from 'react';
+import { createPortal } from 'react-dom';
+import { FocusOn } from 'react-focus-on';
+import TopBar from './TopBar';
+
+const Analysis = () => {
+  const modalPortalElement = document.getElementById('modal-portal');
+  if (!modalPortalElement) return null;
+
+  return createPortal(
+    <Box
+      display="flex"
+      flexDirection="column"
+      w="100%"
+      zIndex="10000"
+      position="fixed"
+      bgColor={colors.background}
+      h="100vh"
+      data-testid="contentBuilderPage"
+    >
```

Comment:
Not sure whether it's something we do, but the layout is pretty well suited for using `grid`, potentially. Did that in the prototype, and made it really flat and easy to play around with moving the various panels.

I guess it's not part of our current palette of styling options, so this definitely works too.

Reply:
We don't use Grid at the moment, at the time we were implementing the `Box` component there was still a requirement to support IE. We can add this as a future Component library improvement but for now, I would keep it as is.

Reply:
Agreed